src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/LoggerMessageDefineTests.cs

```diff
@@ -27,6 +27,12 @@ public async Task CA2253IsProducedForNumericFormatArgument(string format)
             await TriggerCodeAsync(format);
         }
 
+        [Fact]
+        public async Task CA2254ShouldNotApplyForConstantValues()
+        {
+            await TriggerCodeAsync("{|CA1848:logger.LogDebug({|CA2254:$\"{nameof(System.Collections.Generic.IList<object>)}<{{0}}> could not be resovled from DI container, using default JSON binder.\"|}, typeof(string).Assembly)|};");
```

Comment:
```suggestion
            await TriggerCodeAsync("{|CA1848:logger.LogDebug(:$\"{nameof(System.Collections.Generic.IList<object>)}<{{0}}> could not be resovled from DI container, using default JSON binder.\", typeof(string).Assembly)|};");
```

goal is to make test pass after applying this suggestion. Still with this fix it adds `CA2254` for this use case